src/lib/backend.js

```diff
@@ -1,6 +1,6 @@
 // copied from https://github.com/aeternity/superhero-ui/blob/178bc2d63cb362ddc3b2163fed58deb2e253ec00/src/utils/backend.js
 
-const BACKEND_URL = 'https://raendom-backend.z52da5wt.xyz';
+const BACKEND_URL = 'https://test-tipping.aeternity.art';
```

Comment:
could maybe also be set by the user when switching mainnet/testnet

Reply:
We can extend the network configurator to allow the user to set up the config for the tipping functionality (backned, aggregator url, also trusted oracles in the future etc.).

We should have a predefined set of configurations for the current mainnet and testnet builds which are aligned with the network config.

This alone is out of the scope of this PR though and would require some design input.